src/shared/bg_public.h

```diff
@@ -750,7 +750,6 @@ enum dynMenu_t
 {
   MN_NONE,
 
-  MN_WELCOME,
```

Comment:
Deleting this from the list changes the client/server ABI (by renumbering the other ones), so it should be done on a release branch per https://wiki.unvanquished.net/wiki/Compatibility

Reply:
solutions to that point:

* make it clear it's candidate for removal, and remove it for 0.53
* just do the PR against 0.53/sync

About that one, note that I think @Gireen was against removal on IRC (I may be wrong though).

Reply:
I note that from IRC discussions, Gireen don't seem to think resurrecting this is the correct way to proceed.

Reply:
About the ABI change, I've changed the base branch to 0.53.0/sync